.github/workflows/tests-arm64-nightly.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Integration Arm64 Nightly
+permissions: read-all
+on:
+  # schedules always run against the main branch, hence we have to create separate jobs
+  # with individual checkout actions for each of the active release branches
+  schedule:
+    - cron: '30 2 * * *' # runs daily at 2:30 am.
+jobs:
+  main-arm64:
+    uses: ./.github/workflows/tests-arm64-template.yaml
+    with:
+      etcdBranch: main
+      integrationTestCmd: make test-integration
+      unitTestCmd: GO_TEST_FLAGS='-p=2' make test-unit
+  release-35-arm64:
+    uses: ./.github/workflows/tests-arm64-template.yaml
+    with:
+      etcdBranch: release-3.5
+      integrationTestCmd: PASSES='integration' RACE='false' ./test.sh
+      unitTestCmd: PASSES='unit' CPU='4' ./test.sh -p=2
```

Comment:
Could we maybe unify the commands to make it clearer?

Reply:
Can you please clarify.  Do you mean backporting the makefile changes so both `release-3.5` and `main` can use same `make` command?  Currently `release-3.5` seems to need `./test.sh`.

Reply:
Yes, I mean backport the makefile changes to keep a single interface for all branches. It should the forkflows easier to maintain and simplify development of backports. cc @ahrtr

Reply:
I'm happy to create an issue and work on it if we have consensus from others.  I'm not sure I would want it to be a blocking issue for this pr though. Keen to see what overall consensus is 🙂

Reply:
> Yes, I mean backport the makefile changes to keep a single interface for all branches.


Makes sense to me.

Reply:
Created https://github.com/etcd-io/etcd/issues/16176 under our overall ci improvements plan to track.  I am keen to work on this once we have tier 1 arm64 announced 🙏🏻